views/emails/header.ejs

```diff
@@ -0,0 +1,38 @@
+<style>
+    .loginButton {
+      background-color: #000091;
+      color: white;
+      font-size: 1.25rem;
+      line-height: 2rem;
+      min-height: 3.5rem;
+      padding: .75rem 2rem;
+      cursor: pointer;
+      text-decoration: none;
+    }
+  
+    .container {
+      background-color: #f5f5ff;
+      padding: 32px 16px;
+    }
+  
+    .card {
+      background-color: white;
+      margin: auto;
+      max-width: 800px;
+      box-shadow: 0 2px 7px 0 rgb(0 0 0 / 21%);
+    }
+
+    .content {
+      overflow-wrap: break-word;
+      padding: 16px 32px;
+    }
+
+    .image {
+      width: 100%;
+    }
+</style>
+
+<div class="container">
+    <div class="card">
+      <img class="image" alt="Santé Psy Etudiant" src="https://user-images.githubusercontent.com/911434/112502968-d3011980-8d8a-11eb-9e46-f47037a6f809.png">
```

Comment:
Are you sure about the link image? :thinking:

Reply:
it's the same as in the newletter... so yes :/